packages/salesforcedx-vscode-core/src/commands/util/postconditionCheckers.ts

```diff
@@ -284,3 +286,103 @@ export class ConflictDetectionChecker implements PostconditionChecker<string> {
     return { type: 'CONTINUE', data: manifest };
   }
 }
+
+export class TimestampConflictChecker implements PostconditionChecker<string> {
+
+  private isManifest: boolean;
+  private messages: ConflictDetectionMessages;
+
+  constructor(isManifest: boolean, messages: ConflictDetectionMessages) {
+    this.messages = messages;
+    this.isManifest = isManifest;
+  }
+
+  public async check(
+    inputs: ContinueResponse<string> | CancelResponse
+  ): Promise<ContinueResponse<string> | CancelResponse> {
+    if (!sfdxCoreSettings.getConflictDetectionEnabled()) {
+      return inputs;
+    }
+
+    if (inputs.type === 'CONTINUE') {
+      const { username } = workspaceContext;
+      if (!username) {
+        return {
+          type: 'CANCEL',
+          msg: nls.localize('conflict_detect_no_default_username')
+        };
+      }
+
+      const componentPath = inputs.data;
+      const cacheService = new MetadataCacheService(username);
+      const result = await cacheService.loadCache(componentPath, getRootWorkspacePath(), this.isManifest);
+      const detector = new TimestampConflictDetector();
+      const diffs = detector.createDiffs(result);
+      return await this.handleConflicts(inputs.data, username, diffs);
+    }
+    return { type: 'CANCEL' };
+  }
+
+  public async handleConflicts(
+    componentPath: string,
+    usernameOrAlias: string,
+    results: DirectoryDiffResults
+  ): Promise<ContinueResponse<string> | CancelResponse> {
+    const conflictTitle = nls.localize(
+      'conflict_detect_view_root',
+      usernameOrAlias,
+      results.different.size
+    );
+
+    if (results.different.size === 0) {
+      conflictView.visualizeDifferences(conflictTitle, usernameOrAlias, false);
+    } else {
+      channelService.appendLine(
+        nls.localize(
+          'conflict_detect_conflict_header',
+          results.different.size,
+          results.scannedRemote,
+          results.scannedLocal
+        )
+      );
+      results.different.forEach(file => {
+        channelService.appendLine(normalize(file));
+      });
+      channelService.showChannelOutput();
+
+      const choice = await notificationService.showWarningModal(
+        nls.localize(this.messages.warningMessageKey),
+        nls.localize('conflict_detect_override'),
+        nls.localize('conflict_detect_show_conflicts')
+      );
+
+      if (choice === nls.localize('conflict_detect_override')) {
+        conflictView.visualizeDifferences(
+          conflictTitle,
+          usernameOrAlias,
+          false
+        );
+      } else {
+        channelService.appendLine(
+          nls.localize(
+            'conflict_detect_command_hint',
+            this.messages.commandHint(componentPath)
+          )
+        );
+        channelService.showChannelOutput();
```

Comment:
It looks like we're also calling `showChannelOutput` on line 351, do we need to call it again here?

Reply:
I assumed that this method flushed the stream, and therefore was responsible for showing the text that is appended in the line above. Having now tested it, I see that you're right and it's only needed once to switch the channel. Thanks!

Fixed in the latest [commit](https://github.com/forcedotcom/salesforcedx-vscode/pull/3332/commits/59311d19c0602fd852eca39f711ae0ccc3b8f0a3).